src/render/pixie/render_help.py

```
@@ -159,18 +318,31 @@ class HelpRenderer(SimpleCardRenderer):
 
     def __init__(self):
         super().__init__()
+        self._gradient_color = self._pick_coordinated_gradient()
+
+    @classmethod
+    def _pick_coordinated_gradient(cls):
+        """随机选择渐变色，但过滤掉末端颜色过白、在暗色卡片上不协调的渐变"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)